nucypher/network/status/moe.html

```diff
@@ -13,7 +13,7 @@
             window.onload = function () {
                 const ws_port = $ws_port;
                 const origin_hostname = window.location.hostname;
```

Comment:
@KPrasch Do we need to template hostname if it can be passed as a parameter to Moe?